src/ngRoute/route.js

```diff
@@ -615,6 +616,7 @@ function $RouteProvider() {
             }
           }, function(error) {
             if (nextRoute == $route.current) {
+              $log.debug('A $routeChangeError has happened: ' + error);
```

Comment:
Since `error` might be an object, it is better to use `$log.debug('...', error)`.